src/objects/Custom_Column_Header__c.object

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customSettingsType>List</customSettingsType>
+    <enableFeeds>false</enableFeeds>
+    <fields>
+        <fullName>Field_Api_Name__c</fullName>
+        <externalId>false</externalId>
+        <label>Field Api Name</label>
```

Comment:
```suggestion
        <label>Field API Name</label>
```

Reply:
@ptewson-sfdo This also needs Help Text and a Description added. In this case it would be the same for both, "The API name of the field displayed in the list view."